static/usage/v8/badge/inside-tab-bar/angular/example_component_html.md

```diff
@@ -0,0 +1,14 @@
+```html
+<ion-tab-bar color="light">
```

Comment:
Can we update the examples to have 3 tab buttons? Maybe make one not show a badge